docs/src/main/asciidoc/spring-cloud-bus-pubsub.adoc

```diff
@@ -0,0 +1,53 @@
+== Spring Cloud Bus
+
+Using link:https://cloud.google.com/pubsub/[Cloud Pub/Sub] as the link:https://spring.io/projects/spring-cloud-bus[Spring Cloud Bus] implementation is as simple as importing the `spring-cloud-gcp-starter-bus-pubsub` starter.
+
+Maven coordinates, using <<getting-started.adoc#_bill_of_materials, Spring Cloud GCP BOM>>:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.springframework.cloud</groupId>
+  <artifactId>spring-cloud-gcp-starter-bus-pubsub</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source,groovy]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-starter-bus-pubsub'
+}
+----
+
+=== Configuration Management with Spring Cloud Config
```

Comment:
Isn't this Spring Cloud Config + Spring Cloud Bus?

Also, you don't really mention the Spring Cloud Stream binder for Pub/Sub.
All of the configuration for the Pub/Sub binder should apply here too, correct?

Reply:
Spring Cloud Bus is the top level header, but I'll add it here too for searchability.

For stream binder -- I mention it in the sample; I'll add here, too.